packages/contracts/src/dollar/core/ERC1155Ubiquity.sol

```diff
@@ -23,6 +23,22 @@ contract ERC1155Ubiquity is ERC1155, ERC1155Burnable, ERC1155Pausable {
     mapping(address => uint256[]) private _holderBalances;
     uint256 private _totalSupply;
 
+    function accessTotalSupply() public view returns (uint256) {
```

Comment:
Why do we need these methods: `accessTotalSupply()`, `_incrementTotalSupply()`, `_decrementTotalSupply()`?